### ISSUE TITLE
Error when trying to see data content in amplify -> content (Data Manager)

### ISSUE THREAD

**Issue by kNikKumar** [aws-amplify/amplify-category-api#2108]
### Amplify CLI Version

12.2.3

### Question

I tried creating a new table in schema.graphQl :

```
type Stage @model @auth(rules: [{allow: public}]) {
  id: ID!
  name: String!
}

type CommunityType @model @auth(rules: [{allow: public}]) {
  id: ID!
  name: String!
}

type UserType @model @auth(rules: [{allow: public}]) {
  id: ID!
  name: String!
}
```

For all these above data model I am getting below error, when I try to  view it in amplify : 
<img width="861" alt="Screenshot 2023-12-02 at 19 29 31" src="https://github.com/aws-amplify/amplify-category-api/assets/12410523/1f850f3a-6055-44dc-a2a8-f1637da3f382">

I am not able to insert data, Am doing something wrong in schema, amplify push command after creating schema was successful.

**Comment by AnilMaktala** [comment]
Hey @kNikKumar, Thank you for raising this. Are you able to create the data from AppSync console?

**Comment by kNikKumar** [comment]
Hi @AnilMaktala, not for this table. For others yes.

**Comment by AnilMaktala** [comment]
Hey @kNikKumar, Thanks for checking on that. To test something, would you mind updating the UserType model name to something slightly different, like UserType1 or UserTypeV2? Once changed, can you verify and let me know?

**Comment by kNikKumar** [comment]
Also don't know whether it would be helpful or not but, since the something went wrong while table creation, when my app is running I keep getting this warning in terminal :

``` 
WARN  [WARN] 03:08.356 DataStore {"cause": {"data": null, "errors": [[Object], [Object]]}, "errorType": "Unknown", "localModel": null, "message": undefined, "model": "UserType", "operation": null, "process": "sync", "recoverySuggestion": "Ensure app code is up to date, auth directives exist and are correct on each model, and that server-side data has not been invalidated by a schema change. If the problem persists, search for or create an issue: https://github.com/aws-amplify/amplify-js/issues", "remoteModel": null}
```

**Comment by phani-srikar** [comment]
HI @kNikKumar, I don't see anything special with the `UserType` that could cause this issue. Can you check that your `UserTypeTable` is created in your DynamoDB?

**Comment by AnilMaktala** [comment]
Hey @kNikKumar, Are you still experiencing this issue?

**Comment by Maadhav** [comment]
I am facing this issue as well, any solution or a reason for this to happen?

**Comment by kNikKumar** [comment]
Yes still facing this issue, I suspect something is wrong with App Sync, DynamoDB is not correctly syncing up with DataStore . But how to debug this and check, I don't see any error log. 
The only workaround I can think of is create a duplicate Amplify project and start work there.

Is there a way to duplicate my amplify project ?
